server/lib/statsd-collector.js

```diff
@@ -10,9 +10,19 @@ var uaParser = require('ua-parser');
 
 var STATSD_PREFIX = 'fxa.content.';
 
+/**
+ * Normalize the string to make it usable in StatsD tagging.
+ * @param {String} item
+ * @returns {String}
+ * @private
+ */
+function _normalizeTag(item) {
+  return String(item).replace(/ /g, '_').toLowerCase();
```

Comment:
Since the tag name is generated off of the experiment name, and because `:` is reserved, we should guard against experiment names that contain a `:`.